test/node_smoke_tests/commonjs/factory/document_missing.cjs

```diff
@@ -1,15 +1,19 @@
 "use strict";
 
-const assert = require( "node:assert" );
+const main = async() => {
```

Comment:
So far, we have been using function declarations.

```suggestion
async function main() {
```

Reply:
This function no longer exists.